src/Symfony/Component/Notifier/Channel/DesktopChannel.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Notifier\Channel;
+
+use Symfony\Component\Messenger\Exception\ExceptionInterface;
+use Symfony\Component\Notifier\Exception\TransportExceptionInterface;
+use Symfony\Component\Notifier\Message\DesktopMessage;
+use Symfony\Component\Notifier\Notification\DesktopNotificationInterface;
+use Symfony\Component\Notifier\Notification\Notification;
+use Symfony\Component\Notifier\Recipient\RecipientInterface;
+
+/**
+ * @author Ahmed Ghanem <ahmedghanem7361@gmail.com>
+ */
+class DesktopChannel extends AbstractChannel
+{
+    /**
+     * @throws TransportExceptionInterface
+     * @throws ExceptionInterface
```

Comment:
I'm not sure these two lines provide any value, do they?

Reply:
Surely they don't, it's just PhpStorm that always suggests adding those annotations. I've removed them.